### PR TITLE
Adds splash screen guide

### PR DESCRIPTION
**Pull request by rjchaveiro**

### CODE REVIEW

File: splash-screen.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+delete it all together, I suggest deleting it completely.
+
+![delete LaunchScreen](https://cdn-images-1.medium.com/max/1000/1*JX-WHMJ4udPTj79td7avlA.png)
+
+Within the “Images.xcassets” folder there is now a “LaunchImage” option inside of it.
+This is where we’ll drag and drop the iOS images generated for us by Image Gorilla.
+
+![LaunchImage](https://cdn-images-1.medium.com/max/1000/1*0Csm1VYYXEjVhDzXe80vCQ.png)
+
+And that should be it! You may need to close the simulator and JS Packer server then re run `react-native run-ios`
```
Comment (abnersajr) [inline splash-screen.md]: *rerun* not *re run*

File: splash-screen.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Splash screen for React-Native apps
+
+This will be a quick and dirty way to get your app to load the splash screen, based on [this article](https://medium.com/handlebar-labs/how-to-add-a-splash-screen-to-a-react-native-app-ios-and-android-30a3cec835ae) due to the use of automatically generated
+images, it may cause some bloat in the total app size. However it does just do the trick.
+
+## Preparing the images.
+
+First we need the image that we would like to show up when the app opens.
```
Comment (abnersajr) [inline splash-screen.md]: *First,*

File: splash-screen.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Keep in mind that Devices come in all shapes and sizes, therefore our splash
+screen images need to come in a variety of sizes as well.
+For this an easy way to generate the images in the correct sizes is by using [Image Gorilla by Ape tools](http://apetools.webprofusion.com/tools/imagegorilla)
+It's highly recommended to start with a high resolution image, a 2048x2048 px PNG one is what in my opinion works best
```
Comment (abnersajr) [inline splash-screen.md]: *a higher resolution* or *a high-resolution*

File: splash-screen.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+after you are done with submitting the image to Image Gorilla scroll down to the bottom of the page and 'Punch it'.
+When Image Gorilla finishes, download the zip and extract it. There will be multiple directories.
+But the ones that matter are only the iOS and android ones.
+
+## iOS splash screen.
+
+First open the project in xcode
```
Comment (abnersajr) [inline splash-screen.md]: First,

File: splash-screen.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# Splash screen for React-Native apps
+
+This will be a quick and dirty way to get your app to load the splash screen, based on [this article](https://medium.com/handlebar-labs/how-to-add-a-splash-screen-to-a-react-native-app-ios-and-android-30a3cec835ae) due to the use of automatically generated
```
Comment (willamesoares) [inline splash-screen.md]: Add a period after `this article` and start a new sentence `... this article. Due to the use...`

File: splash-screen.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+images, it may cause some bloat in the total app size. However it does just do the trick.
```
Comment (willamesoares) [inline splash-screen.md]: The last sentence would sound better like this: `However, it will do the trick.`
However, IMHO this sentence could be just removed.

File: splash-screen.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+## Preparing the images.
+
+First we need the image that we would like to show up when the app opens.
+Keep in mind that Devices come in all shapes and sizes, therefore our splash
```
Comment (willamesoares) [inline splash-screen.md]: `devices`

File: splash-screen.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+screen images need to come in a variety of sizes as well.
+For this an easy way to generate the images in the correct sizes is by using [Image Gorilla by Ape tools](http://apetools.webprofusion.com/tools/imagegorilla)
```
Comment (willamesoares) [inline splash-screen.md]: `For this, ...`

File: splash-screen.md (hunk 3 above)
Comment (willamesoares) [inline splash-screen.md]: It reads better if you end this sentence after `... high-resolution image.` and then start another one for the remaining part. Also, for the sake of the context, it is not good to write `in my opinion`, since this is not directly related to someone. This would fit better in a blog post for instance, where you can point whose that opinion is.

File: splash-screen.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+It's highly recommended to start with a high resolution image, a 2048x2048 px PNG one is what in my opinion works best
+after you are done with submitting the image to Image Gorilla scroll down to the bottom of the page and 'Punch it'.
```
Comment (willamesoares) [inline splash-screen.md]: Comma after `Gorilla`

File: splash-screen.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+First open the project in xcode
+```
+open ios/<YourProjectName>.xcodeproj
+```
+
+In the main window of your project at the general tab, scroll down until you see 'App Icons and Launch Images'
+
+![xcode](https://cdn-images-1.medium.com/max/1000/1*UjtDz2Kjyq-8Wv4KkI8EJw.png)
+
+Press the 'Use asset catalog...' button for a selection dialog to appear, select 'LaunchImage' and press 'Migrate'
```
Comment (willamesoares) [inline splash-screen.md]: `Click on the 'Use asset catalog' button and select the 'LaunchImage' option in the dialog box. Then, click on the 'Migrate' button.`

File: splash-screen.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+Then on the left side of xcode, in the project navigator, expand the first folder with your project's name.
```
Comment (willamesoares) [inline splash-screen.md]: `Then, ... `

File: splash-screen.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+Inside there you will find a file named `LaunchScreen.xib` you may either open it and remove all of it' content or
```
Comment (willamesoares) [inline splash-screen.md]: `Inside that folder, you will find a file named LaunchScreen.xib. You may either open it and remove all of its content or delete it all together. I suggest deleting it completely.`

File: splash-screen.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+delete it all together, I suggest deleting it completely.
+
+![delete LaunchScreen](https://cdn-images-1.medium.com/max/1000/1*JX-WHMJ4udPTj79td7avlA.png)
+
+Within the “Images.xcassets” folder there is now a “LaunchImage” option inside of it.
```
Comment (willamesoares) [inline splash-screen.md]: You can remove the `inside of it` at the end.

File: splash-screen.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+This is where we’ll drag and drop the iOS images generated for us by Image Gorilla.
```
Comment (willamesoares) [inline splash-screen.md]: `we'll`

File: splash-screen.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+
+![LaunchImage](https://cdn-images-1.medium.com/max/1000/1*0Csm1VYYXEjVhDzXe80vCQ.png)
+
+And that should be it! You may need to close the simulator and JS Packer server then re run `react-native run-ios`
+to see the splash Screen.
```
Comment (willamesoares) [inline splash-screen.md]: `screen`

File: splash-screen.md (hunk 4 above)
Comment (boniattirodrigo) [inline splash-screen.md]: Xcode

File: splash-screen.md (hunk 11 above)
Comment (boniattirodrigo) [inline splash-screen.md]: Xcode

File: splash-screen.md (hunk 10 above)
Comment (rjchaveiro) [inline splash-screen.md]: IMO using `for a selection dialog to appear` gives a more clear notion to the reader following the tutorial on what to expect. Telling them to "select something in the dialog box" doesn't properly convey where the next action should take place.

Comment (willamesoares) [inline splash-screen.md]: Well, I think it is pretty intuitive to tell the user to select that option in the dialog box since this is what is going to show up when he/she clicks on the `Use asset catalog` button, regardless of what he/she is expecting. There is no other dialog box and the `LaunchImage` option is going to be there, so the user will know which dialog box and which option you are referring to.

This was just a suggestion, you can keep in that way and maybe add some gifs simulating the steps as it has been done in other parts of the documentation :smile:
